build-release-tools/application/release_npm_packages.py

```diff
@@ -0,0 +1,230 @@
+#!/usr/bin/env python
+# Copyright 2016, DELLEMC, Inc.
+
+"""
+This is a command line program that makes a rackhd release to npmjs.
+
+Usage:
+./on-build-config/build-release-tools/HWIMO-BUILD on-build-config/build-release-tools/application/release_npm_packages.py \
+--build-directory b/ \
+--manifest-file rackhd-devel \
+--git-credential https://github.com,GITHUB \
+--npm-credential registry.npmjs.org,6a2d8de2-dfce-4f59-a866-20f1baa394bd \
```

Comment:
It looks like those credentials are for a personal test account, and not the actual `rackhd` one, though.